presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -748,6 +748,34 @@ public void testGreatCircleDistance()
         assertInvalidFunction("great_circle_distance(36.12, -86.67, 33.94, infinity())", "Longitude must be between -180 and 180");
     }
 
+    @Test
+    public void testSTInteriorRings()
+    {
+        assertInvalidFunction("transform(ST_InteriorRings(ST_GeometryFromText('POINT (2 3)')), x -> ST_ASText(x))",
```

Comment:
let's make a couple of changes to improve readability;

* drop `transform`
* extract helper method `assertInvalidInteriorRings`

```
    assertInvalidInteriorRings("POINT (2 3)", "POINT");
    
    private void assertInvalidInteriorRings(String wkt, String geometryType)
    {
        assertInvalidFunction(format("ST_InteriorRings(ST_GeometryFromText('%s'))", wkt), format("ST_InteriorRings only applies to POLYGON. Input type is: %s", geometryType));
    }
```